### DISCUSSION TITLE
Alpine Vaultwarden LXC does not update.

### DISCUSSION THREAD

**Discussion by mguedes71** [tteck/Proxmox#2235] [Help Needed] [answered]
Alpine Vaultwarden LXC does not update. Despite having an indication that it has updates (Web and Server), it does not update when I use the script indicated for this purpose (bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ ct/alpine-vaultwarden.sh)"). Any help?

![image](https://github.com/tteck/Proxmox/assets/34092794/939f2bb9-0d1d-4bea-bd25-bc7b1b791e5a)

**Comment by tteck** [comment, marked as answer]
Depends which repository you're using...
https://pkgs.alpinelinux.org/packages?name=vaultwarden&branch=edge&repo=&arch=x86_64&maintainer=

**Reply by mguedes71** [reply]
You're right! Thank you, I will provide instructions for you to update the repo. Merry Christmas!

**Reply by ncodee** [reply]
> Depends which repository you're using... https://pkgs.alpinelinux.org/packages?name=vaultwarden&branch=edge&repo=&arch=x86_64&maintainer=

I'd imagine the `edge` branch contains the latest release? If so, there doesn't seem to be any packages for 1.32.2 found [here](https://github.com/dani-garcia/vaultwarden/releases).

**Reply by tteck** [reply]
With Alpine packages, you're at the mercy of the maintainer.
